content/gsoc/gsoc-2024/_index.md

```diff
@@ -57,8 +56,4 @@ Will update soon.
 12. We have ex-GSoC students actively serving on our Board of Directors and in Executive Officer positions, so GSoC has definitely proven to be a great career path for students wanting to get more involved in open source and information security R&D.
 13. Honeypots and honeynet technology, research and tools have filtered down benefits to many areas of IT, web development, operational service management, Internet education and computer security research
 
-### Other ways to get involved
```

Comment:
With cleaned up i didn't mean removed but the links need to be fixed to point to the correct places on the website :)